feeds/wlan-ap/opensync/src/platform/openwrt/src/netifd/src/wifi_inet_config.c

```diff
@@ -156,17 +155,24 @@ static void wifi_inet_conf_load(struct uci_section *s)
 	if (tb[NET_ATTR_IFNAME])
 		ifname = blobmsg_get_string(tb[NET_ATTR_IFNAME]);
 
-	if (ifname && *ifname == '@')
-		SCHEMA_SET_STR(conf.parent_ifname, &ifname[1]);
-	else if (strcmp(conf.if_type, "gre"))
-		dhcp_get_config(&conf);
+	if (ifname) {
+		if(!strncmp(ifname, "gre4", strlen("gre4")) || !strncmp(ifname, "gre6", strlen("gre6")))
```

Comment:
Here you appear to be handling gre6 which I think is IPv6 tunnel.  However, earlier, you were only retrieving the IPv4 address of the WAN interface.  Is this an issue?

Reply:
GRE on IPv6 is not fully supported yet. Keeping the code as much as possible so that we have to do minimum changes when we are ready.

Reply:
Thanks